src/Security/MemberAuthenticator/CookieAuthenticationHandler.php

```diff
@@ -176,16 +176,20 @@ public function authenticateRequest(HTTPRequest $request)
 
         // Renew the token
         $rememberLoginHash->renew();
-        $tokenExpiryDays = RememberLoginHash::config()->uninherited('token_expiry_days');
-        Cookie::set(
-            $this->getTokenCookieName(),
-            $member->ID . ':' . $rememberLoginHash->getToken(),
-            $tokenExpiryDays,
-            null,
-            null,
-            false,
-            true
-        );
+
+        // Send the new token to the client if it was changed
+        if ($rememberLoginHash->getToken()) {
```

Comment:
Can you please remind me why we need this new condition?

Reply:
`getToken()` outputs the unhashed version of the token, which is only available during the request that generates it; If we don't regenerate the token, we can't (and don't need to) retransmit this value.